pyroute2/iproute/linux.py

```diff
@@ -60,6 +60,14 @@
     RTM_NEWTCLASS,
     RTM_NEWTFILTER,
     RTM_SETLINK,
+    RTMGRP_IPV4_IFADDR,
+    RTMGRP_IPV4_ROUTE,
+    RTMGRP_IPV4_RULE,
+    RTMGRP_IPV6_IFADDR,
+    RTMGRP_IPV6_ROUTE,
+    RTMGRP_IPV6_RULE,
```

Comment:
I do not found RTMGRP_IPV6_RULE constant in current kernel headers. Is it a typo with RTNLGRP_IPV6_RULE ? (RTM vs RTN)

Reply:
Ok, it looks like all RTMGRP_* variables are outdated and working only as compat.

See https://lwn.net/Articles/147608/ for reference

Reply:
There is already `NetlinkSocket.add_membership()/drop_membership()`, so probably I have to change NDB sources to use it. Let's schedule it as a separate task.